test/hotel_controller_test.rb

```diff
@@ -6,26 +6,23 @@
     @date = Date.parse("2020-08-04")
   end
   describe "wave 1" do
-    describe "rooms" do
```

Comment:
You should keep this test to ensure that a `hotel_controller` has an array of rooms.